doc/source/conf.py

```diff
@@ -19,27 +15,87 @@
 # import os
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
+import pyspectral.version as current_version
```

Comment:
Copy-paste typo? Maybe this should be from `trollflow2` :wink:

Reply:
Oh, baad, thx @pnuu !
Will fix...